.github/workflows/ci.yml

```diff
@@ -12,5 +12,21 @@ permissions:
   contents: read
 
 jobs:
-  Shared:
-    uses: fog/.github/.github/workflows/ci.yml@v1.4.0
```

Comment:
Once this has settled, I think we'll want to go back to the shared action probably, so just noting that here so I don't forget.